src/Toolkit/Toolkit.Maui/BasemapGallery/BasemapGallery.Appearance.cs

```diff
@@ -48,8 +49,25 @@ static BasemapGallery()
             outerScrimContainer.RowDefinitions.Add(new RowDefinition { Height = new GridLength(40) });
             outerScrimContainer.RowSpacing = 4;
 
-            Grid thumbnailGrid = new Grid() { WidthRequest = 64, HeightRequest = 64 };
+            Grid thumbnailGrid = new Grid() { WidthRequest = 64, HeightRequest = 64 };
+#if ANDROID
+            // Workaround for .NET MAUI Android bug where CollectionView throws
+            // Java.Lang.RuntimeException: 'Canvas: trying to use a recycled bitmap'
+            // (see https://github.com/dotnet/maui/issues/11519, affects MAUI 9.0.0).

```

Comment:
That bug was supposed to be [fixed](https://github.com/dotnet/maui/pull/12310) in .NET 7.

I wonder if MAUI team's fix regressed, or was incomplete.  I see that similar issued have been reported many times, including a [.NET 8 regression issue](https://github.com/dotnet/maui/issues/18834) that was just closed for inactivity.

If it's easy to repro in our toolkit, perhaps we should report it to the MAUI team to fix.  It seems that https://github.com/dotnet/maui/issues/23373 is the most-recent active issue.